Automattic/jetpack
# Shortcodes: remove Hulu

#### Changes proposed in this Pull Request:

**Hulu** - This is no longer a service that has embedding features. [View their post](https://twitter.com/hulu_support/status/1283562756726300672). We have also deprecated the Hulu block in Gutenberg

Following the previously deprecated codes, this deletes the PHP files and adds both to the `unavailable` function.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Create a post
* Add a shortcodes block `[hulu]`
* Observe the notification

<img width="702" alt="Markup 2021-09-17 at 00 15 03" src="https://user-images.githubusercontent.com/33258733/133693700-d472202b-d189-44d4-b81e-b25121f86167.png">
